### ISSUE TITLE
High-distance lamps that respect obstructions

### ISSUE THREAD

**Issue by seancrain** [DcZipPL/lightestlampsmod#9]
**Is your feature request related to a problem? Please describe.**
I need to use the high-powered lamps to cover as much ground as possible, but in addition to this, I need these lamps to respect solid walls that would block normal lights.

**Describe the solution you'd like**
I would like to see a variant of the omega (and lower variant) lamps that respect light obstructions. If this is already possible, please let me know how I can make them.

**Describe alternatives you've considered**
There aren't any alternatives available for 1.15 as far as I know

**Additional context**
I think the description is self-explanatory.

**Comment by DcZipPL** [comment]
Making lamps that respect solid walls can be hard. I need to make pathfinding algorithm but I don't have much time to work on it.
I don't close the issue. **Thanks for suggestion!**

**Comment by Jergeric** [comment]
yeah like the quantum lights mod (https://github.com/DarkMorford/Quantum-Lights) from 1.12.2, thats was great.
I would be really very grateful for that, because currently I have whole biomes illuminated 😅
